igcollect/ping.py

```diff
@@ -4,83 +4,90 @@
 Copyright (c) 2020 InnoGames GmbH
 """
 
-import _thread
+# this script is used to collect ping data to evaluate availability and latency
+# of our network (this should replace smokeping in the longrun)
+#
+# Please Not that the package fping has to be installed on your Linux server!
+
+
 import argparse
 import subprocess
+import time
+
+from multiprocessing import Pool
+from os import cpu_count
 
-from time import time, sleep
 
 def parse_args():
     parser = argparse.ArgumentParser(
-        description='Ping colecctor script for Graphana ')
-    parser.add_argument('adress', metavar='URL', type=str, nargs='+',
-                        help='the address zu ping to')
+        description='Ping collector script for Graphite ')
+    parser.add_argument('hosts', metavar='h', nargs='+',
+                        help='the adresses to ping')
     parser.add_argument('-c', '--count', type=int, default=20,
-                        help='how many tims should the script tries to ping')
-    parser.add_argument('--prefix', default='ping',
-                        help='the path to the value in Graphana')
+                        help='how many times should the script tries to ping')
```

Comment:
This is a question and not a statement:
```suggestion
                        help='how many times the script will try to ping')
```